kubernetes.js

```diff
@@ -343,6 +343,14 @@ const createIngress = async (project, options) => {
     localIngress.spec.rules[0].host = url.host
     localIngress.spec.rules[0].http.paths[0].backend.service.name = `${prefix}${project.safeName}`
 
+    if (url.protocol === 'https') {
+        localIngress.spec.tls = [{
+            hosts: [
+                url.host
+            ]
+        }]
```

Comment:
secretName: "ff-apps-ca-secret"

Or something, to hold the wildcard ca certificate to be used for applications, would need to be documented.

Reply:
`ff-apps-ca-secrets` is not a Secret, it's a collection of CA certs that the instance should trust, so not useful here.

I'm still working out how to solve this nicely, there are 2 possible approaches

1. A single shared wildcard cert/key in a single secret
2. A secret per instance, and something like CertManager issuing new ACME provisioned certs for each.

Both have pros/cons for different environments.